appveyor.yml

```diff
@@ -22,7 +18,7 @@ configuration:
   - Release
 
 build_script:
-  - ps: .\scripts\BuildAndTest.ps1 -NoTest -NoPackage -NoPublish -NoSigningDirectory -Configuration Release
+  - ps: .\scripts\BuildAndTest.ps1 -NoBuildSample -NoTest -NoPackage -NoPublish -NoSigningDirectory -Configuration Release
```

Comment:
>NoBuildSample [](start = 36, length = 13)

We can't yet build the sample on AppVeyor because the invocation of `NuGet.exe restore` causes a null reference exception that I haven't yet been able to debug. But we _can_ (and do) build it on the desktop.